cider/exec/nextgen/operators/RowToColumnNode.cpp

```diff
@@ -97,6 +92,32 @@ class ColumnWriter {
     expr_->setLocalIndex(local_offset);
   }
 
+  JITValuePointer setFixSizeRawData(utils::FixSizeJITExprValue& fixsize_val) {
+    if (expr_->get_type_info().get_type() == kBOOLEAN) {
+      auto raw_data_buffer = context_.getJITFunction()->createLocalJITValue(
+          [this]() { return allocateBitwiseBuffer(1); });
+      // leverage existing set_null_vector but need opposite value as input
+      // TODO: (yma11) need check in UT
+      context_.getJITFunction()->emitRuntimeFunctionCall(
+          "set_null_vector",
```

Comment:
Could use ```cider_agg_id_proj_bool```.

Reply:
`cider_agg_id_proj_bool` take `int8_t` as the value and a cast from 1bit is needed if call this method.